packages/solo/src/init-basedir.js

```diff
@@ -71,29 +65,6 @@ export default function initBasedir(
     fs.writeFileSync(path.join(dstHtmldir, gr), revision);
   }
 
```

Comment:
Yay not copying the vat files any more.. we changed swingset to include a copy of the vat sources in the kernelDB, so this hasn't been necessary for stability across reboots in a long time, but I never got around to removing the code that copied those files into the base directory that we originally relied upon. Thanks for removing it!